nsfailover.sh

```diff
@@ -39,16 +39,16 @@
 
 # Environment variables
 [ -z "${LOG_LEVEL}" ]       && LOG_LEVEL="6" # 7 = debug, 0 = emergency
-[ -z "${NS_ENABLE}" ]       && NS_ENABLE="no" # Set to no to disable
-[ -z "${NS_TESTDOMAIN}" ]   && NS_TESTDOMAIN="google.com" # Use this to determine if NS is healthy
-[ -z "${NS_1}" ]            && NS_1="" # Primary Nameserver (172.16.0.23 for Amazon EC2). You need to set this yourself
-[ -z "${NS_2}" ]            && NS_2="8.8.8.8" # Secundary Nameserver: Google
-[ -z "${NS_3}" ]            && NS_3="4.2.2.2" # Tertiary Nameserver: Level3
+[ -z "${NS_ENABLE}" ]       && NS_ENABLE="yes" # Set to no to disable
+[ -z "${NS_TESTDOMAIN}" ]   && NS_TESTDOMAIN="servicos.nc.ufpr.br" # Use this to determine if NS is healthy
```

Comment:
seems like some unrelated changes leaked into the pr

Reply:
Yes. Clean in commit 
805f8b0